docs/changelog.md

```diff
@@ -1394,7 +1394,7 @@ Bugfixes and other improvements for your Zulip experience.
 ### Highlights for users (since 26.1.124 / 26.2.125)
 
 * Highlight colors for code blocks now match the webapp and
-  offer more contrast, especially in night mode.
+  offer more contrast, especially in dark theme.
```

Comment:
This is a historical changelog entry (from 2019), so let's leave it in the form we used at the time.

Reply:
Okay